src/components/HeaderTodoApp/HeaderTodoApp.tsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, {
+  FC, FormEvent, useState,
+} from 'react';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  setError: (error: string) => void;
+}
+
+export const HeaderTodoApp: FC<Props> = React.memo(({
+  todos,
+  setError,
+}) => {
+  const [query, setQuery] = useState('');
+
+  const handleSubmit = async (event: FormEvent) => {
+    event.preventDefault();
+
+    if (!query) {
+      setError("Title can't be empty");
+
+      return;
+    }
+
+    setQuery('');
```

Comment:
I know it's not main goal of this task but you can set here actual data from query and not just an empty line